src/Transfer/Transfer.php

```diff
@@ -0,0 +1,389 @@
+<?php
+
+/**
+ * Transfer class.
+ *
+ * @package EightshiftForms\Transfer
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Transfer;
+
+use EightshiftForms\CustomPostType\Forms;
+use EightshiftForms\Helpers\UploadHelper;
+use WP_Query;
+
+/**
+ * Transfer class.
+ */
+class Transfer implements TransferInterface
+{
+	/**
+	 * Use trait Upload_Helper inside class.
+	 */
+	use UploadHelper;
+
+	/**
+	 * Type global settings key.
+	 */
+	public const TYPE_GLOBAL_SETTINGS = 'globalSettings';
+
+	/**
+	 * Type forms key.
+	 */
+	public const TYPE_FORMS = 'forms';
+
+	/**
+	 * Export global settings.
+	 *
+	 * @return array<int, array<string, mixed>>
+	 */
+	public function getExportGlobalSettings(): array
+	{
+		global $wpdb;
+
+		$options = $wpdb->get_results( // phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching
+			"SELECT option_name as name, option_value as value
+				FROM $wpdb->options
+				WHERE option_name REGEXP 'es-forms-'"
```

Comment:
```suggestion
				WHERE option_name LIKE '%es-forms-%'"
```
Small perf optimisation